app/javascript/utils/fetch-json.ts

```diff
@@ -0,0 +1,27 @@
+import { camelizeKeys } from 'humps'
```

Comment:
Maybe [camelcase-keys](https://github.com/sindresorhus/camelcase-keys#readme) is a better choice given the author / maintenance-state.

Reply:
I tried using this, but it doesn't work.

Reply:
Did you give it `{ deep: true }` as option?